.travis.yml

```diff
@@ -1,15 +1,29 @@
-language: node_js
-node_js: "stable"
+matrix:
+  include:
+    - language: node_js
+      node_js: "stable"
 
-before_install:
-  - npm install -g npm@latest
+      before_install:
+      - npm install -g npm@latest
 
-install:
-  - npm install -g htmlhint
-  - npm install -g jshint
-  - npm install -g stylelint
+      install:
+      - npm install -g htmlhint
+      - npm install -g jshint
+      - npm install -g stylelint
 
-script:
-  - htmlhint **/*.html *.html --config htmlhint.conf
-  - jshint ./js/*.js --config .jshint
-  - stylelint "css/*.css" --config .stylelintrc.json
+      script:
+        - htmlhint **/*.html *.html --config htmlhint.conf
+        - jshint ./js/*.js --config .jshint
+        - stylelint "css/*.css" --config .stylelintrc.json
+
+    - language: python
+      python:  3.6
+      cache: pip
+      install:
+        - pip install pylinkvalidator
+      script:
+        - python -m http.server 8001 &> /dev/null & pid=$!
+        - sleep 5
```

Comment:
``` sleep 5 ``` so that the server gets the time to start being trying to connect and ``` kill ${pid} ``` to end everything as all the tests have been done.